src/shared/amplify.config.ts

```diff
@@ -38,9 +61,18 @@ export const signUpConfig = {
       required: true,
       displayOrder: 2,
       type: "password",
+      custom: false,
```

Comment:
Can you add a comment to explain what custom do ? and a link to the documentation for signUpFields if it's not there already.